src/notebook.ts

```diff
@@ -6,49 +6,20 @@ import * as path from 'path';
 import * as fs from 'fs';
 import * as os from 'os';
 
-// This function looks in the user's default extensions folder for Azure Data Studio
-// to find this extension and its packaged files. If it encounters an error, a
-// message will appear in an error window.
-const processNotebooks = () => {
-    const rootExtensionsFolder = path.normalize(path.join(os.homedir(), '.azuredatastudio', 'extensions'))
-    let notebookNames: Array<string> = [];
-
-    let subExtensionFolder = getFolderContent(rootExtensionsFolder);
-
-    subExtensionFolder.forEach(folderName => {
-        findCorrectFolder(folderName, rootExtensionsFolder, notebookNames);
-    });
-    return notebookNames;
-}
-
-// This function is called by processNotebooks to find the correct folder that contains
-// this extension. If it is found, then it opens up the book in Azure Data Studio's
-// native notebook viewlet.
-const findCorrectFolder = (folderName: string, rootExtensionsFolder: string, notebookNames: Array<string>) => {
-    let folderExt = path.basename(folderName).toLowerCase();
-
-    if (folderExt.indexOf(('vasubhog.dot-net-interactive-kernels').toLowerCase()) > -1) {
-        let fullFolderPath = path.join(rootExtensionsFolder, folderName, 'content');
-        try {
-            extractNotebooksFromFolder(fullFolderPath, notebookNames);
-        } catch (e) {
-            vscode.window.showErrorMessage("Unable to access " + fullFolderPath + ": " + e.message);
-        }
+// This function looks in the user's extensions folder for notebooks to be opened.

```

Comment:
I'd suggest updating this to indicate that it looks specifically in the folder for this extension